src/main/java/gov/usgs/cida/pubs/webservice/CustomAsyncExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package gov.usgs.cida.pubs.webservice;
+
+import java.lang.reflect.Method;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.aop.interceptor.AsyncUncaughtExceptionHandler;
+
+public class CustomAsyncExceptionHandler implements AsyncUncaughtExceptionHandler {
+	private static final Logger LOG = LoggerFactory.getLogger(CustomAsyncExceptionHandler.class);
+
+	@Override
+	public void handleUncaughtException(Throwable throwable, Method method, Object... obj) {
+		LOG.error("Exception message - " + throwable.getMessage());
```

Comment:
Maybe being a little too pedantic here. It seems that someone looking at the log, would not have any context as
to what is going on. Maybe add a log line above such as  "Task execution failed:"

Otherwise, looks good.